src/solver/types.jl

```diff
@@ -241,6 +248,7 @@ Base.@kwdef struct AtmosModel{
     vert_diff::VD = nothing
     viscous_sponge::VS = nothing
     rayleigh_sponge::RS = nothing
+    prognostic_surface_model::PS = nothing
```

Comment:
```suggestion
    surface_model::SM = nothing
```
I think we can just call it `surface_model`, and it is either `PrescribedSurfaceTemperature` or `PrognosticSurfaceTemperature`. @dennisYatunin What do you think?